KristianJBorgwarth/Golferino
# Lms/feat/extend player

This pull request does the following:

*Removals*
- Removed player_model.py
- Removed player_created_event and related code
- removed PasswordService, VertificationService

*Introductions*
- creating "User" (builtin django User) / auth_user in db. This automatically handles password hashing
- Introduced Change of Password in a viewer
- Reintroduced Email verification. This time, however, email is verified by visitng the URL that is sent by email - no verification code
- added authorization as a prerequisite to query playerrounds. Authorization is done by logging in to an existing user in the swagger browser (Upper right corner "Authorize")

*Test?*
- Create a user
- Ensure it's verified (set to true manually through DBMS
- Authorize with that user in swagger
- create a playerrounds
- query that playerround
- use the password_change_update endpoint to change the password
- try to logout of the authorizer and query playerround again - see the return msg.
